<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qt_vlcClass</class>
 <widget class="QMainWindow" name="Qt_vlcClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt_vlc</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame_playerAll">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" rowstretch="8,2" columnstretch="8,2">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <property name="spacing">
        <number>1</number>
       </property>
       <item row="0" column="0">
        <widget class="QWidget" name="wiget_player_window" native="true">
         <layout class="QGridLayout" name="gridLayout"/>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QListWidget" name="listWidget_TvStation"/>
       </item>
       <item row="1" column="0">
        <widget class="QSlider" name="playerSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1055</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_operation">
    <property name="title">
     <string>控制</string>
    </property>
   </widget>
   <addaction name="menu_operation"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
